Cargo.toml

```diff
@@ -22,6 +22,7 @@ clap = "3.0.13"
 mysql = "22.0.0"
 log = "0.4.14"
 env_logger = "0.9.0"
+execute = "*"
```

Comment:
I think this is not used anymore. Please remove.

Reply:
I deleted execute.